### PR TITLE
Add table flag

### PR DESCRIPTION
**Pull request by duc0905**
Create --table flag to print outputs in table format
Use Awsmock for testing
Change callback functions in services.js to run properly
Github link: https://github.com/duc0905/nodecloud-cli/tree/add_table_flag

### CODE REVIEW

File: lib/helper.js [hunk 1]
```diff
@@ -30,16 +31,127 @@ let logofied = text => {
   );
 };
 
+let tablerForArray = data => {
+  if (!Array.isArray(data)) {
+    return "";
+  }
+  let finalTable = new Table({
+    chars: {
+      top: "═",
```
Comment (JaDogg) [inline lib/helper.js]: Can you extract these configs to a constant instead of duplicating it across both functions?

File: lib/helper.js [hunk 2]
```diff
@@ -30,16 +31,127 @@ let logofied = text => {
...
+  let body = [];
+
+  // making the head of the table
+  for (prop in data) head.push(prop);
+  finalTable.push(head);
+
+  // making the body of the table
+  for (prop in data) {
+    if (typeof data[prop] == "object") {
+      if (Array.isArray(data[prop])) body.push(tablerForArray(data[prop]));
```
Comment (JaDogg) [inline lib/helper.js]: Can you use curly braces here. This is unreadable.

File: lib/helper.js [hunk 3]
```diff
@@ -30,16 +31,127 @@ let logofied = text => {
...
+  return finalTable.toString();
+};
+
+let tabler = data => {
+  if (Array.isArray(data)) {
+    return tablerForArray(data);
+  } else if (typeof data == "object") {
+    return tablerForObject(data);
+  } else {
+    console.log(
```
Comment (JaDogg) [inline lib/helper.js]: Throw an error?

**Review by JaDogg — CHANGES_REQUESTED** [review]

File: lib/helper.js (hunk 3 above)
Comment (duc0905) [reply lib/helper.js]: Yes I think I should do so.

File: lib/helper.js (hunk 2 above)
Comment (duc0905) [reply lib/helper.js]: Ok, thank you for your advice

**Pull request closed without merging**